src/main/scala/tilelink/Broadcast.scala

```diff
@@ -28,6 +34,94 @@ case class TLBroadcastParams(
   require (numTrackers > 0)
 }
 
+/** [[TLBroadcast]] is the last level coherence manger for TileLink. Simplest topology can be this:
+  * {{{
+  *   A           B
+  *   │           │
+  *   └─────┬─────┘
+  *         │
+  *     Broadcast
+  *         │
+  *   ┌─────┴─────┐
+  *   │           │
+  *   A'          B'
+  *   └─────┬─────┘
+  *       XBar
+  *         │
+  *         C
+  * }}}
+  *
+  * It probes clients cachelines to maintain coherence.
+  * It can store or not store directory, depending on the implementation of [[ProbeFilter]] in this Broadcast.
+  *
+  * @example Here is some example transactions on the broadcast.
+  *          if A is Tip, B acquire this cacheline:
+  *          - (t0) B -> BH: AcquireBlock
+  *          - (t1) BH -> A: ProbeBlock
+  *          - (t1) A -> BH: ProbeAckData/ProbeAck
+  *          -   (t2) if ProbeAckData: BH -> C: PutFullData
+  *          -   (t2) if ProbeAckData: C -> BH: AccessAck
+  *          - (t3) BH -> C: Get
+  *          - (t3) C -> BH: AccessAckData
+  *          - (t0) BH -> B: GrantData
+  *          - (t0) B -> BH: GrantAck
```

Comment:
Maybe use "M1" and "M2"  instead of "A" and "B"? Since A and B are also TL priority channel names I think it is a little confusing.